bottles/frontend/ui/details-bottle.blp

```diff
@@ -230,6 +215,142 @@ template DetailsBottle : .AdwPreferencesPage {
 
   .AdwPreferencesGroup group_programs {
     title: _("Programs");
+
+    .AdwActionRow row_no_programs {
+      Box {
+        hexpand: true;
+
+        Label {
+          label: _("Use the Installers section or the \"Run executable\" button.");
+          wrap: false;
+
+          styles [
+            "dim-label",
+          ]
+        }
+      }
+    }
+    
+    .AdwPreferencesRow bottom_bar {
+      activatable: false;      
+      
+      Box {
+        height-request:45;
+        
+        Button add_shortcuts {
+          hexpand: true;
+          has-frame: false;
+          focusable: false;
+          Box {
+            halign: center;
+            Image {
+              icon-name: "list-add-symbolic";
+            }
+            
+            Separator {
+              styles [
+                "spacer"
+              ]
+            }
+            
+            Label {
+              label: _("Add Shortcuts...");
+            }
+          }
+        }
+        
+        Separator {
+        }
+        
+        Button add_programs {
+          hexpand: true;
+          has-frame: false;
+          Box {
+            halign: center;
+            Image {
+              icon-name: "list-add-symbolic";
+            }
+          
+            Separator {
+              styles [
+                "spacer"
+              ]
+            }
+
+            Label {
+              label: _("Add Programs...");
+            }
+          }
+        }
+        
+        styles [
+          "linked"
+        ]
+      }
+    }
+  }
+
+  .AdwPreferencesGroup group_options {
+    title: _("Options");
+      
+      .AdwActionRow row_preferences {
+        activatable: true;
+        title: _("Preferences");
+        subtitle: _("Some random text here.");
+
+        Image {
+          icon-name: "go-next-symbolic";
+        }
+      }
+      
+      .AdwActionRow row_dependencies {
+        activatable: true;
+        title: _("Dependencies");
+        subtitle: _("Improve compatibility of Windows software.");
+
+        Image {
+          icon-name: "go-next-symbolic";
+        }
+      }
+      
+      .AdwActionRow row_installers {
+        activatable: true;
+        title: _("Installers");
+        subtitle: _("Install community curated programs.");
+
+        Image {
+          icon-name: "go-next-symbolic";
+        }
+      }
+
+      .AdwActionRow row_programs {
+        activatable: true;
+        title: _("Programs");
+        subtitle: _("Manage installed programs.");
+
+        Image {
+          icon-name: "go-next-symbolic";
+        }
+      }
+
+      .AdwActionRow row_snapshots {
+        activatable: true;
+        title: _("Snapshots");
+        subtitle: _("Imagine windows restore points, but better.");
```

Comment:
not helpful nor descriptive :)

Reply:
Texts are not final in this PR yet, i just put whatever for now :)